tests/README.md

```diff
@@ -0,0 +1,68 @@
+# pvc-migrate Tests
+
+This is a collection of tests for pvc-migrate.
+
+## Pre-Requisites
+
+### Prepare Kubeconfigs for Source and Destination clusters
+
+Create source and destination kubeconfigs in the `tests` directory.
+
+Login to your Source (3.x) cluster and save the kubeconfig in `src.config` file:
+
+```sh
+touch src.config && export KUBECONFIG=src.config
+
+oc login -u <username> -p <password> <src_cluster_api_url>
+```
+
+Login to your Destination (4.x) cluster and save the kubeconfig in `dest.config` file:
+
+```sh
+touch dest.config && export KUBECONFIG=dest.config
+
+oc login -u <username> -p <password> <destination_cluster_api_url>
+```
+
+### Prepare test configuration file
+
+Copy `test-config.yml.sample` to `test-config.yml`:
+
+```sh
```

Comment:
nitpicking but please add

```
cd tests/
```

Reply:
The readme is already under tests/ directory. If someone is reading it, they are already in tests directory